src/test/typescript/typescript-parser_test.ts

```diff
@@ -0,0 +1,183 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+// import * as fs from 'fs';
+// import * as path from 'path';
+import * as ts from 'typescript';
+
+import stripIndent = require('strip-indent');
+
+import {Analyzer} from '../../analyzer';
+import {UrlLoader} from '../../url-loader/url-loader';
+// import {FSUrlLoader} from '../../url-loader/fs-url-loader';
+import {UrlResolver} from '../../url-loader/url-resolver';
+import {ParsedTypeScriptDocument} from '../../typescript/typescript-document';
+import {TypeScriptParser} from '../../typescript/typescript-parser';
+
+class TestUrlResolver implements UrlResolver {
```

Comment:
Why define a resolver and loader for this test? The simpler style of the other parser tests seems better. Can use the on-disk fixtures and exercise a normal analyzer flow